src/views/upload/technicalQualification/wamitabQualification.html

```diff
@@ -20,10 +20,13 @@
       <div id="has-no-annotations" class="form-group">
         <div id="wamitab-qualification-description">
           <p>
-            Check the <a target="_blank" rel="noopener noreferrer" href="https://wamitab.org.uk/competence/operator-competence/">{{> abbreviations/WAMITAB }} risk tables (opens new tab)</a> to find out which qualification you need for this permit. If the qualification is over 2 years old you also need a continuing competence certificate.
+            <span class="bold-small">Upload copies of the certificates.</span>
```

Comment:
ID and unit test needed

Reply:
The dynamic component of this which is whether the div with id="wamitab-qualification-description" exists or not is already covered in the "when there are annotations" and "when there are no annotations" tests in the uploadHelper.  An option of descrptionId = "wamitab-qualification-description" is passed in within the wamitabQualification.route.test.  I would argue that tests written to test all the hard coded text within the html within the page would render the tests brittle.